examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -1045,7 +1045,7 @@ def main(args):
         pipeline = pipeline.to(accelerator.device)
 
         # load attention processors
-        pipeline.load_attn_procs(args.output_dir)
+        pipeline.load_lora_weights(args.output_dir)
```

Comment:
We should be using the more general method here.

Reply:
Agree!